doc/classes/Camera2D.xml

```diff
@@ -324,18 +324,24 @@
 	</methods>
 	<members>
 		<member name="anchor_mode" type="int" setter="set_anchor_mode" getter="get_anchor_mode" enum="Camera2D.AnchorMode">
+			The Camera2D's anchor point. See [code]ANCHOR_MODE_*[/code] constants.
 		</member>
 		<member name="current" type="bool" setter="_set_current" getter="is_current">
+			Makes the camera  the current for the scene(viewport and layer)
```

Comment:
Extra space: "camera  the"

Needs a bit more explanation, and booleans have a prescribed format per the guidelines: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

    If [code]true[/code] this camera is the active camera for the current scene. Only one camera can be current, so setting a different camera [code]true[/code] will disable this one.

Reply:
I've done some tests to see the behavior of the current setter and I think it would be good to inform that in case of two or more cameras setted to true, the last one will remain current while the others will turn false... 
something like:
'Only one camera can be current, so setting a different camera [code]true[/code] will make all other cameras [code]false[/code].' 

Or is it overexplaining?
Thanks